### ISSUE TITLE
JDK 11 Ubuntu path

### ISSUE THREAD

**Issue by VladislavAntonyuk** [appveyor/ci#2815]
What is the path to jdk-11 on Ubuntu?

**Comment by Wasapl** [comment]
Its in `/usr/lib/jvm/java-11-openjdk-amd64/`
There is `JAVA_HOME_11_X64` variable which contain path to jdk-11.
Also in `appveyor.yml` you can use [stack](https://www.appveyor.com/docs/getting-started-with-appveyor-for-linux/#configuring-language-stack) directive to switch to jdk-11:
```
stack: jdk 11
```

**Comment by VladislavAntonyuk** [comment]
where should I put stack?
  -
    matrix:
      only:
        - image: Ubuntu
    environment:
      PLATFORM: Linux
    stack: jdk 11
    build_script:
      - sh: >-
          sudo apt update

**Comment by Wasapl** [comment]
Yes, your example should work

**Comment by VladislavAntonyuk** [comment]
does stack directive work with windows? or it is just linux feature?

**Comment by Wasapl** [comment]
Yes, right now this is only Linux feature. But right now we're working on unifying Windows and Linux experience.
